packages/@aws-cdk/aws-ec2/lib/vpc.ts

```diff
@@ -1177,7 +1177,7 @@ export class Vpc extends VpcBase {
    */
   private subnetConfiguration: SubnetConfiguration[] = [];
 
```

Comment:
At a glance, it looks like many of these can be applied to master instead of `v2-main`.